libraries/core-react/src/components/Checkbox/Checkbox.tsx

```diff
@@ -16,12 +16,15 @@ const StyledCheckbox = styled.label<StyledCheckboxProps>`
 `
 
 const LabelText = styled.span`
-  ${typographyTemplate(tokens.typography)}
+  ${typographyTemplate(tokens.typography)};
+  ${spacingsTemplate(tokens.entities.label.spacings)}
 `
 
 export type CheckboxProps = {
-  /** Label for the checkbox */
-  label: string
+  /** Label for the checkbox
+   * Do _not_ leave this blank unless you are using `aria-labelledby` instead!
```

Comment:
its also valid to use `aria-label`... I suggest we remove this reminder as its something thats part of the documentation or guidelines. Property descriptions should be short and descriptive of the property

Reply:
Removed 🔥